docs/content/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -175,11 +175,10 @@ mapping can be inverted with the following interface.
 
 ```go
 
-// AnalyzeObjectID examines an object in a kube-bind service provider
-// cluster and returns the object's ID as known in the service consumer
-// cluster and the kube-bind ID for the consumer, or an error if
-// the object does not appear to be a provider's copy of a consumer's object.
-func AnalyzeObjectID(obj metav1.Object) (namespace, name, kbSpaceID string, err error) {..}
+// AnalyzeClusterScopedObject examines a cluster-scoped object in a kube-bind service provider cluster
+// and returns the object's ID as known in the service consumer cluster and the kube-bind ID for the
```

Comment:
Now that we do not need a term for Namespace+Name, we can simply say "object name" (which is a well-defined technical term) instead of "object ID" (which is less so).

Reply:
done